source/includes/steps-configure-custom-function-auth-realm-ui.yaml

```
@@ -51,36 +51,42 @@ content: |
      :alt: The function selection dropdown in the Realm UI.
      :width: 750px
      :lightbox:
-  
+
   .. example::
-     
+
      An application implements a Custom Function authentication provider
      that stores user data in the ``app.users`` MongoDB collection. The
      app lets users log in by specifying their ``username`` but does not
      require a password or any other type of authentication.
-     
-     The appication's authentication function queries the ``users``
+
+     The application's authentication function queries the ``app.users``
      collection for an existing user with the specified username. If the
      user already exists, the function returns their stored ``id`` value. If
      the user does not exist, the function stores a new user document in
      the collection and returns that document's ``id`` value.
-     
+
+     .. note::
+       The collection ``app.users`` is not created by Realm. You can use
```

nit: avoid passive voice ("is not created") when possible. Active voice, in comparison, helps users feel more engaged with text because the objects you describe actually "act" on other objects instead of just potentially acting.
If you decide you'd like to reword this, you could use something like this (feel free to customize this as much as you'd like; this is just a suggestion!):
```suggestion
       {+service-short+} does not automatically create an ``app.users`` collection for your application. You can use
```

---

PS: after we renamed "Stitch" into "Realm" (and realized that "Realm" also includes the concept of "realms"), we decided that we should save ourselves some work if the product ever gets renamed again, so we now use [source constants](https://docs.mongodb.com/meta/reference/source-constants/) to replace the a few key terms in the documentation. You can see the full list of replacements [here](https://github.com/mongodb/docs-realm/blob/master/snooty.toml#L4) and [here](https://github.com/mongodb/docs-realm/blob/master/conf.py#L72) (and yes, you do have to update both places to manually keep them in sync -- I know it sucks, but it's an artifact of our transition from oldgen to nextgen).